tests/science/global_param.classic.4.2_ecflux.txt

```diff
@@ -0,0 +1,106 @@
+FORCE_FORMAT    ASCII
+FORCE_ENDIAN    LITTLE
+N_TYPES     12
+FORCE_TYPE  SKIP
+FORCE_TYPE  SKIP
+FORCE_TYPE  SKIP
+FORCE_TYPE  SKIP
+FORCE_TYPE  PREC       UNSIGNED 40
+FORCE_TYPE  AIR_TEMP   SIGNED   100
+FORCE_TYPE  SHORTWAVE  SIGNED   100
+FORCE_TYPE  LONGWAVE   SIGNED   100
+FORCE_TYPE  SKIP
+FORCE_TYPE  PRESSURE   SIGNED   100
+FORCE_TYPE  VP         SIGNED   100
+FORCE_TYPE  WIND       SIGNED   100
+FORCE_DT    1
+FORCEYEAR   1980
+FORCEMONTH  01
+FORCEDAY    01
+FORCEHOUR   00
+GRID_DECIMAL    4
+
+WIND_H          10.0
+MEASURE_H       2.0
+ALMA_INPUT  FALSE
+
+SOIL            $test_data_dir/datasets/ec_flux_towers/params/soil_param.site_test.txt
+ARC_SOIL        FALSE
+BASEFLOW    ARNO
+JULY_TAVG_SUPPLIED  FALSE
+ORGANIC_FRACT   FALSE
+
+VEGLIB          $test_data_dir/datasets/ec_flux_towers/params/veg_lib_IGBP.fv
+VEGLIB_VEGCOVER TRUE
+# VEGLIB_IRR      FALSE
+
+VEGPARAM        $test_data_dir/datasets/ec_flux_towers/params/veg_param.sites.IGBP.modis
+ROOT_ZONES      3
+
+VEGPARAM_LAI    TRUE
+VEGPARAM_VEGCOVER TRUE
+VEGPARAM_ALB    TRUE
+
+LAI_SRC         FROM_VEGPARAM
+VEGCOVER_SRC    FROM_VEGPARAM
+ALBEDO_SRC      FROM_VEGPARAM
+SNOW_BAND   1
+
+
+RESULT_DIR      $result_dir
+OUT_STEP        1
+SKIPYEAR        0
+COMPRESS        FALSE
+BINARY_OUTPUT   FALSE
+ALMA_OUTPUT     FALSE
+MOISTFRACT      FALSE
+PRT_HEADER      TRUE
+PRT_SNOW_BAND   FALSE
+
+N_OUTFILES  2
+
+OUTFILE         snow 16
+OUTVAR          OUT_SWE                 %.4f    *       *
+OUTVAR          OUT_SNOW_DEPTH          %.4f    *       *
+OUTVAR          OUT_SNOW_CANOPY         %.4f    *       *
+OUTVAR          OUT_SNOW_COVER          %.4f    *       *
+OUTVAR          OUT_ADVECTION           %.4f    *       *
+OUTVAR          OUT_DELTACC             %.4f    *       *
+OUTVAR          OUT_SNOW_FLUX           %.4f    *       *
+OUTVAR          OUT_RFRZ_ENERGY         %.4f    *       *
+OUTVAR          OUT_MELT_ENERGY         %.4f    *       *
+OUTVAR          OUT_ADV_SENS            %.4f    *       *
+OUTVAR          OUT_LATENT_SUB          %.4f    *       *
+OUTVAR          OUT_SNOW_SURF_TEMP      %.4f    *       *
+OUTVAR          OUT_SNOW_PACK_TEMP      %.4f    *       *
+OUTVAR          OUT_SNOW_MELT           %.4f    *       *
+OUTVAR          OUT_ALBEDO              %.4f    *       *
+OUTVAR          OUT_SALBEDO             %.4f    *       *
+
+OUTFILE         en_bal 10
+OUTVAR          OUT_ADV_SENS            %.4f    *       *
+OUTVAR          OUT_ADVECTION           %.4f    *       *
+OUTVAR          OUT_DELTAH              %.4f    *       *
+OUTVAR          OUT_GRND_FLUX           %.4f    *       *
+OUTVAR          OUT_IN_LONG             %.4f    *       *
+OUTVAR          OUT_LATENT              %.4f    *       *
+OUTVAR          OUT_LATENT_SUB          %.4f    *       *
+OUTVAR          OUT_R_NET               %.4f    *       *
+OUTVAR          OUT_SENSIBLE            %.4f    *       *
+OUTVAR          OUT_RAD_TEMP            %.4f    *       *
```

Comment:
suggestion: change format from `%.4f` to `%.6g` or something with a bit more precision.